pkg/master/master_test.go

```diff
@@ -1001,7 +1001,7 @@ func testInstallThirdPartyResourceRemove(t *testing.T, version string) {
 	}
 
 	path := makeThirdPartyPath("company.com")
-	master.RemoveThirdPartyResource(path)
+	master.RemoveThirdPartyResource(path + "/foos")
```

Comment:
makeThirdPartyPath appears to produce a path that ends in a group, therefore it needs a version before a type should appear.

Also, path.Join (the path package, that is) is better for this operation.